Making it easier for our users to report documentation issues or questions (#522)
argonne-lcf/user-guides: docs/issues.md

# Questions/Issues on ALCF Docs

> Our ALCF documentations are hosted through GitHub to make it easier to contribute and collaborate.

Reviewer: ```suggestion
The ALCF documentation source code is hosted on GitHub to make it easier to contribute and collaborate.
```

> If you find any issues on our documentation page, raise an issue at [ALCF Github user guide webpage](https://github.com/argonne-lcf/user-guides/issues/new). Provide us with a brief description on the documentation issue, page url, section heading and mark the appropriate label type on the right pane. 

Reviewer: ```suggestion
If you find any issues within these pages, please open an Issue at [ALCF GitHub user guide webpage](https://github.com/argonne-lcf/user-guides/issues/new). Provide us with a brief description of the issue, page URL, section heading, and mark the appropriate label type on the right pane. 
```
